### ISSUE TITLE
Student metrics

### ISSUE THREAD

**Issue by jzthompson** [maltyeva/mindimp-pg#12]
Hi Matt and Maria,

Wanted to add some ideas for how metrics could be organized on the website. Maybe you've already thought of this, but I suggest that we think of metrics as being organized under two main categories:

1. Student performance - We evaluate students' performance in courses and other programs. This information feeds into the grades and certifications that we disseminate to students, but we should also develop indicators based on student performance to help evaluate our own programs.

2. Student and parent feedback - We'd gather these stats from client surveys at the ends of courses and at other key times. This information would just be for us. We'd use it to improve our services and programs.

Basically, we assess students and they assess us. Both types of data can help us improve. I'll be interested to hear your thoughts and see what you guys put together.
